src/main/java/avd/inf/jdm/rentmycar/controller/OfferController.java

```diff
@@ -134,7 +138,33 @@ public ResponseEntity<Object> create(@RequestBody OfferDTO offerDTO) {
                 }
             }
 
-            Offer newOffer = offerService.create(offerDTO.getStartDateTime(), offerDTO.getEndDateTime(), offerDTO.getPickupLocation(), car);
+            // Get the Latitude and Longitude of the pickup location, using http://api.positionstack.com/v1/forward?access_key=04e13b946c065785becd602f43c14ece&query=
+            String url = "http://api.positionstack.com/v1/forward?access_key=04e13b946c065785becd602f43c14ece&query=" + offerDTO.getPickupLocation();
```

Comment:
de accesskey in een .env variable zou beter zijn, maar is prima voor nu..

Reply:
Helemaal mee eens. Ik wilde het gewoon snel af hebben en had tussendoor mijn API-key al gepushed naar de Repo, dus hij was toch al openbaar en toen dacht ik, ik laat het wel zo en refactor dit later wel netjes 🤣